src/index.js

```diff
@@ -14,23 +14,40 @@ export const dataPointsEndpointUrl = (project = PROJECT) => (
   `${TREEHERDER}/api/project/${project}/performance/data/`
 );
 
-export const subtestsPerfDataUrl =
+export const perfDataUrls =
   (frameworkId, signatureIds, project = PROJECT, interval = DEFAULT_TIMERANGE) => {
     const url = dataPointsEndpointUrl(project);
-    const queryParams = stringify({
+    const baseParams = stringify({
       framework: frameworkId,
       interval,
-      signature_id: signatureIds,
     });
-    return `${url}?${queryParams}`;
+    const urls = [];
+    for (let i = 0; i < (signatureIds.length) / 100; i += 1) {
+      const signaturesParams = stringify({
+        signature_id: signatureIds.slice(i * 100, ((i + 1) * 100)),
+      });
+      urls.push(`${url}?${baseParams}&${signaturesParams}`);
```

Comment:
The logic here is a bit hard to understand. Maybe consider using lodash's chunk here? That's what we did in perfherder's compare view

https://lodash.com/docs/4.17.10#chunk